internal/alb/tg/targethealth.go

```diff
@@ -76,12 +80,20 @@ type targetHealthController struct {
 // SyncTargetsForReconciliation starts a go routine for reconciling pod condition statuses for the given targets in the background until they are healthy in the target group
 func (c *targetHealthController) SyncTargetsForReconciliation(ctx context.Context, t *Targets, desiredTargets []*elbv2.TargetDescription) error {
 	conditionType := podConditionTypeForIngressBackend(t.Ingress, t.Backend)
-
 	targetsToReconcile, err := c.filterTargetsNeedingReconciliation(conditionType, t, desiredTargets)
 	if err != nil {
 		return err
 	}
-
+	// use static condition type if no targets to reconcile with the old type
```

Comment:
Please try to avoid using "old and new type" in the comments as this merely refers to the history of implementation and makes it more confusing to read.

Reply:
ok.